test/units/testsuite-71.sh

```diff
@@ -89,10 +89,50 @@ test_chassis() {
     fi
 }
 
+restore_sysfs_dmi() {
+    umount /sys/class/dmi/id
+    rm -rf /run/systemd/system/systemd-hostnamed.service.d
+    systemctl daemon-reload
+    systemctl stop systemd-hostnamed
+}
+
+test_firmware_date() {
+    trap restore_sysfs_dmi RETURN
+
+    # Ignore /sys being mounted as tmpfs
+    mkdir -p /run/systemd/system/systemd-hostnamed.service.d/
+    cat >/run/systemd/system/systemd-hostnamed.service.d/override.conf <<EOF
+[Service]
+Environment="SYSTEMD_DEVICE_VERIFY_SYSFS=0"
+EOF
+    systemctl daemon-reload
+
+    # No DMI on s390x or ppc
+    if [[ ! -d /sys/class/dmi/id ]]; then
```

Comment:
Not a big matter, but the check should be done earlier than creating drop-in config and `systemctl daemon-reload`. Then, testing on s390 or ppc slightly become faster.